tests/src/common.hh

```diff
@@ -81,7 +82,7 @@ struct CornerFactory {
 template <int gridDim>
 struct ValidCornerFactory {
   static void construct(std::vector<Dune::FieldVector<double, gridDim>>& values, const Dune::GeometryType& type,
-                        const bool& isRandomlyDistorted) {
+                        const bool& isRandomlyDistorted, const bool& isFixedDistorted = false) {
```

Comment:
Can you wrap these to bools into an enum type:

`enum class CornerDistorationFlag
{
 unDistorted,
fixedDistorted,
randomlyDistorted
}; 

and decide from this. (It will clean up the code a bit)